sessionctx/variable/varsutil.go

```diff
@@ -128,7 +128,7 @@ func TiDBOptOn(opt string) bool {
 	return strings.EqualFold(opt, "ON") || opt == "1"
 }
 
-func tidbOptPositiveInt(opt string, defaultVal int) int {
+func tidbOptInt32(opt string, defaultVal int) int {
```

Comment:
I think the old name is more specific, maybe we should revert this name refactor ?

Reply:
it has another function called `tidbOptInt64`

Reply:
`tidbOptInt64` doesn't check whether this values is positive, while `tidbOptPositiveInt` does.

Reply:
maybe `tidbOptPositiveInt32` is a good name ?